docs/deploy-cephfs.md

```diff
@@ -75,7 +75,7 @@ you're running it inside a k8s cluster and find the config itself).
 | `csi.storage.k8s.io/provisioner-secret-namespace`, `csi.storage.k8s.io/node-stage-secret-namespace` | for Kubernetes | Namespaces of the above Secret objects                                                                                                                                                                                  |
 | `encrypted`                                                                                         | no             | disabled by default, use `"true"` to enable fscrypt encryption on PVC and `"false"` to disable it. **Do not change for existing storageclasses**                                                                          |
 | `encryptionKMSID`                                                                                   | no             | required if encryption is enabled and a kms is used to store passphrases                                                                                                                                                |
-
+| `extraDeploy`                                                                                       | no             | array of extra objects to deploy with the release |
```

Comment:
I guess you need to keep an empty line after the table. The MarkDown linter complains about the table, but that shouldn't be the issue.

Reply:
Thx, fixed